index.html

```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Web3 Teams</title>
+</head>
+
+<body>
+
+    <section class="mixed">
+        <header>
+            <nav class="mobile-nav">
+                <div class="mobile-header">
+                    <a href="#"> <img src="./images/hamburger-icon.png" alt="hamburger icon" id="open-btn"></a>
+                </div>
+            </nav>
+            <!-- dropdown -->
+            <div class="side-nav" id="side-nav">
+                <a href="#" class="close-btn" id="close-btn"><img src="./images/cross-button.png"
+                        alt="cross button"></a>
+                <ul>
+                    <li>
+                        <a href="./about.html">About</a>
+                    </li>
+                    <li>
+                        <a href="#">Program</a>
+                    </li>
+                    <li>
+                        <a href="#">Join</a>
+                    </li>
+                    <li>
+                        <a href="#">Sponsor</a>
+                    </li>
```

Comment:
- [ ] It will be more user friendly if you can add appropriate `href `attributes for the individual links. 👍 
- [optional] Can we have a `link` to the contact section as well? Thanks.

> Don't forget to implement it for both desktop and mobile versions. 💯